Makefile

```diff
@@ -65,6 +65,7 @@ guess_platform_script := \
 guess_platform := $(shell $(guess_platform_script))
 
 all: all-$(guess_platform)
+check: check-$(guess_platform)
 
```

Comment:
I was think that that would allow you to run `make check-linux-x86_64` and `make check-linux-x86` and get all the tests both times.